### DISCUSSION TITLE
「求助」MAC 平台新装的 pag 导出插件，AE崩溃

### DISCUSSION THREAD

**Discussion by doubleYang1020** [Tencent/libpag#2427] [General] [answered]
![image](https://github.com/user-attachments/assets/fd67a365-2a7e-41c6-b2d5-8f56cd7869fa)

**Comment by doubleYang1020** [comment]
我这边尝试过 用pagviewer 卸载插件 然后重新安装也没有用，基本环境信息，mac 平台，14.1.1系统，M1 pro cpu，ae 版本2024

**Comment by doubleYang1020** [comment]
我这边临时解决了，应该是软件冲突问题
之前在这台电脑上安装过Insta360 STUDIO https://www.insta360.com/cn/download
这个软件在AE插件目录Adobe/Common/Plug-ins/7.0/MediaCore下有个Insta360Importer.bundle
![image](https://github.com/user-attachments/assets/e3ac9409-b985-40f2-a37f-ffc27bbfe4bc)
我把这个删除后，pag 导出插件可正常使用了。
虽然问题解决了，还请帮忙看看是什么原因。

**Comment by jinwuwu001** [comment]
您好，该问题我在如下环境下未复现
Apple M3 Pro，macOS 15.3
AE 2024/2025 
Insta360 STUDIO 5.5.0
PAGViewer 4.3.6.102
检查目录/Library/Application\ Support/Adobe/Common/Plug-ins/7.0/MediaCore/下并没有安装Insta360Importer.bundle，这块环境有差异。
请问该问题是否还能复现，若能请描述下如何安装Insta360Importer.bundle

**Reply by doubleYang1020** [reply]
https://www.insta360.com/cn/download/hot-download，安装Insta360 STUDIO

**Reply by jinwuwu001** [reply]
上述回复中已说明我本地安装Insta360 STUDIO 5.5.0后未出现该问题，是否有稳定复现的路径的，是否和Insta360 STUDIO版本有关

**Comment by MolixEs77777** [comment]
Intel芯片，系统：MacOS Sonoma 14.5, 出现同样问题，检查目录/Library/Application\ Support/Adobe/Common/Plug-ins/7.0/MediaCore/下并没有安装Insta360Importer.bundle ，移除其他渲染器测试，仍旧崩溃

**Reply by jinwuwu001** [reply]
请描述下具体的复现环境，包括PAGViewer、AE版本等信息
另外可从官网下载安装最新版PAGViewer和导出插件验证是否还有类似问题
https://pag.io/

**Reply by MolixEs77777** [reply]
PAGView 是最新的，ae 2019-2025所有版本都试过了，不行

**Reply by jinwuwu001** [reply]
好的，我在inter芯片的电脑上复现了该问题，我们近期会排查下问题原因

**Reply by MolixEs77777** [reply]
感谢，修好了请滴滴我一声，再次感谢

**Reply by jinwuwu001** [reply, marked as answer]
该问题已修复，请从官网下载4.3.158版本使用
https://pag.io/docs/pag-download.html

**Reply by MolixEs77777** [reply]
<img width="695" alt="image" src="https://github.com/user-attachments/assets/cd68dd27-cf0b-47eb-8c3e-58898068fbfe" />
还是要崩溃啊...下了4.3.158 卸了插件又重新安装了

**Reply by MolixEs77777** [reply]
2025.6.5，我从官网下了最新插件，崩溃问题解决了
